interfaces/certificate_transfer/v0/schemas/requirer.json

```diff
@@ -5,7 +5,18 @@
   "title": "`certificate_transfer` requirer schema",
   "description": "The `certificate_transfer` root schema comprises the entire requirer databag for this interface.",
   "default": {},
+  "examples": [
+    {
+      "version": 0
+    }
+  ],
   "properties": {
-    "additionalProperties": true
-  }
+    "version": {
+      "$id": "#/properties/version",
+      "type": "integer",
+      "title": "Interface version",
+      "description": "Highest supported version of the interface by this requirer"
+    }
```

Comment:
Same comment here, the version number says nothing about other potential supported versions. It's simply the version of the interface that is being used now.

Reply:
Fixed